internal/runner/runner.go

```diff
@@ -257,21 +264,86 @@ func (r *Runner) Run() error {
 	r.startWorkers()
 
 	r.wgresolveworkers.Wait()
+	if r.stats != nil {
+		r.stats.Stop()
```

Comment:
Maybe I'm misunderstanding, but I guess the stats should be running if not nil?

Reply:
Yes, the `r.stats` variable would be not nil if user has provided the `-stats` flag. After the resolving dns process completed, the stats would only display 100% not so it is a good idea to stop it completely.

Reply:
The purpose of this piece of code is to stop dnsx from printing this to output after the resolving dns process completed:

`[0:01:05] | Hosts: 700 | RPS: 10 | Requests: 700/700 (100%)`
`[0:01:10] | Hosts: 700 | RPS: 9 | Requests: 700/700 (100%)`
`[0:01:15] | Hosts: 700 | RPS: 8 | Requests: 700/700 (100%)`
...
Many more until the filtering wildcard subdomains are completed.